### PR TITLE
Add test for customers in queue

### PR DESCRIPTION
**Pull request by stringtheory1234**

### CODE REVIEW

File: backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java [hunk 1]
```diff
@@ -161,6 +152,29 @@ public void testGetShop_success() {
         assertEquals("Shop Type ", SHOP_TYPE, newShop.getShopType());
     }
 
+    @Test
+    public void testCustomersInQueue_success() {
+        // Arrange.
+        UUID shopId = addShopToRepository(SHOP_OWNER_ID, SHOP_NAME, SHOP_ADDRESS, PHONE_NUMBER, SHOP_TYPE);
+        addTokenListToShop(shopId);
+
```
Comment (KarishmaSinghChauhan) [inline backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java:160]: Nit. Is this an extra newline here?

Comment (stringtheory1234) [reply backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java:160]: yup removed it!

File: backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java [hunk 2]
```diff
@@ -161,6 +152,28 @@ public void testGetShop_success() {
...
+        // Act.
+        long customersInQueue = inMemoryVirtualQueueRepository.getCustomersInQueue(shopId);
+
+        // Assert.
+        assertEquals("Customers in Queue is ", inMemoryVirtualQueueRepository.getShopIdToListOfTokensMap().get(shopId).stream().filter(token -> token.getStatus() == Status.ACTIVE).count(), customersInQueue);
+    }
+
+    private UUID addTokenToTokenMap() {
```
Comment (priytanvani) [inline backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java:168]: refer to comment regarding this in karishma's pr.

File: backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java [hunk 3]
```diff
@@ -120,13 +120,13 @@ public void testGetTokensForShop_success() throws Exception {
 
         // Assert.
         assertEquals("Size of token list", getTokensResponseList.size(),
-                inMemoryVirtualQueueRepository.getShopIdToListOfTokensMap().get(shopId).size());
+                inMemoryVirtualQueueRepository.getShopIdToListOfTokensMap().get(shopId).stream().filter(token -> token.getStatus() == Status.ACTIVE).count());
```
Comment (priytanvani) [inline backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java:123]: Reverse the order here and in other tests applicable?
expected value and then actual value that we got in the act block?

File: backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java [hunk 4]
```diff
@@ -173,6 +186,8 @@ private UUID addShopToRepository(String shopOwnerId, String shopName, String sho
     private void addTokenListToShop(UUID shopId) {
         List<Token> tokens = new ArrayList<>();
         tokens.add(new Token(UUID.randomUUID(), shopId, 1, Status.ACTIVE));
+        tokens.add(new Token(UUID.randomUUID(), shopId, 1, Status.EXPIRED));
```
Comment (priytanvani) [inline backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java:189]: No need to change now, but would recommend you to read up on Builder pattern.

Comment (stringtheory1234) [reply backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java:189]: okay will definitely look into that!

File: backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java (hunk 3 above)
Comment (stringtheory1234) [reply backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java]: done!

**Pull request merged**